integrationTests/testObserverConnector.go

```diff
@@ -0,0 +1,143 @@
+package integrationTests
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"time"
+
+	wsData "github.com/multiversx/mx-chain-communication-go/websocket/data"
+	wsFactory "github.com/multiversx/mx-chain-communication-go/websocket/factory"
+	"github.com/multiversx/mx-chain-core-go/data/outport"
+	"github.com/multiversx/mx-chain-core-go/marshal"
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/multiversx/mx-chain-notifier-go/api/shared"
+	"github.com/multiversx/mx-chain-notifier-go/common"
+	"github.com/multiversx/mx-chain-notifier-go/config"
+	"github.com/multiversx/mx-chain-notifier-go/factory"
+	"github.com/multiversx/mx-chain-notifier-go/process"
+)
+
+// CreateObserverConnector will create observer connector component
+func CreateObserverConnector(facade shared.FacadeHandler, connType string, apiType string) (ObserverConnector, error) {
```

Comment:
Should the rest of the funcs except for this be exported?

Reply:
no, unexported them